includes/class-evf-form-handler.php

```diff
@@ -94,12 +95,13 @@ public function delete( $ids = array() ) {
 	 * Create new form.
 	 *
 	 * @since  1.0.0
-	 * @param  string $title
-	 * @param  array  $args
-	 * @param  array  $data
-	 * @return mixed
+	 * @param  string $title    Form title.
+	 * @param  string $template Form template.
+	 * @param  array  $args     Form Arguments.
+	 * @param  array  $data     Additional data.
+	 * @return int|bool Form ID on successful creation else false.
 	 */
-	public static function create( $title = '', $template = 'blank', $args = array(), $data = array() ) {
+	public function create( $title = '', $template = 'blank', $args = array(), $data = array() ) {
```

Comment:
This need to be strictly checked on addons because I have to remove static call here seeing the core scope.